projects/plugins/jetpack/modules/woocommerce-analytics/class-jetpack-woocommerce-analytics.php

```diff
@@ -76,6 +76,12 @@ public static function should_track_store() {
 		if ( ! $minimum_woocommerce_active ) {
 			return false;
 		}
+
+		// Do not track if the user has opted out of tracking.
+		if ( 'yes' !== get_option( 'woocommerce_allow_tracking', 'no' ) ) {
```

Comment:
Nitpic: What would be the advantage of using
```php
get_option( 'woocommerce_allow_tracking', 'no' )
```
instead of
```php
get_option( 'woocommerce_allow_tracking' )
```

It's my understanding that `woocommerce_allow_tracking` can only have the values `yes` and `no`. I'd probably leave the default value out, as I do not see in which way it would add value here.

Reply:
Both solutions will work indeed. I added it in there so it's clearer what values we can expect, but I'm happy to remove the default value, I agree it doesn't add a ton of value here.

Reply:
Done in 8a1ce9da6bf3627aaac7d680ea081f04b513799d